_faq/80_developers.md

```diff
@@ -22,7 +22,7 @@ A: We encourage you to read [Translating JabRef](https://github.com/JabRef/jabre
 ## Q: I want to send a patch. Where can I do that?
 
 A: We really prefer the pull request approach of GitHub.
-In case you want to contribute something, but git seems to be too hard for you, please get in touch with the developers to work out other ways of contributing.
+In case you want to contribute something, but git seems to be too hard for you, please get in touch with the developers to work out other ways of contributing (jabrefmail+webfeedback@gmail.com).
```

Comment:
Note that we already have:
>Q: I am a programmer. How can I join the JabRef project?
>
>A: We follow the usual GitHub pull request approach. We have a lot of open issues. If you want to be sure to work on something the core developers currently do not handle, check the issues tagged with asking-for-a-pull-request. See our CONTRIBUTING.md for detailed information and further links.

If we keep the question, shall we say patches are not accepted?